src/main/java/org/mskcc/kickoff/upload/JiraFileUploader.java

```diff
@@ -217,87 +243,40 @@ private JiraUser getPmJiraUser(String projectManagerIgoName) {
         return pmJiraUserRetriever.retrieve(projectManagerIgoName);
     }
 
-    private IssueStatus retrieveJiraIssueState(String requestId) {
-        Issue issue = getIssue(requestId);
 
+    private IssueStatus retrieveJiraIssueState(String key) {
+        Issue issue = getIssueByKey(key);
         String currentState = issue.getStatus().getName();
         return statusFactory.getStatus(currentState);
     }
 
-    public void uploadFiles(KickoffRequest kickoffRequest, String requestId) {
-        for (ManifestFile manifestFile : ManifestFile.getRequiredFiles()) {
-            if (manifestFile.isFileGenerated())
-                uploadSingleFile(kickoffRequest, manifestFile, requestId);
-        }
-    }
-
-    public void changeStatus(String transitionName, String issueId) {
-        Issue issue = getIssue(issueId);
-        String previousStatus = issue.getStatus().getName();
-
+    @Override
+    public void changeStatus(String transitionName, String key) {
+        Issue issue = getIssueByKey(key);
         Promise<Iterable<Transition>> transitionsPromise = restClient.getIssueClient().getTransitions(issue);
+
         Iterable<Transition> transitions = transitionsPromise.claim();
 
         for (Transition transition : transitions) {
             String name = transition.getName();
             if (transitionName.equalsIgnoreCase(name)) {
-                Promise<Void> setGeneratedStatus = restClient.getIssueClient().transition(issue, new TransitionInput
-                        (transition.getId()));
+                Promise<Void> setGeneratedStatus = restClient.getIssueClient().transition(issue, new TransitionInput(transition.getId()));
                 setGeneratedStatus.claim();
-
-                issue = getIssue(issueId);
-                String newStatus = issue.getStatus().getName();
-
-                LOGGER.info(String.format("Status for issue: %s changed from: \"%s\" to: \"%s\" using transition: " +
-                        "%s", issue
-                        .getSummary(), previousStatus, newStatus, name));
-
-                return;
+                LOGGER.info(String.format("Status for issue: %s changed to: %s", issue.getSummary(), name));
+                break;
             }
         }
-
-        throw new NoTransitionFoundException(String.format("No transition found with name %s for issue %s for current" +
```

Comment:
in case an attempt to make transition which doesn't exists we won't be informed about this as no exception will be thrown and no warning will be logged I think